_projects/area_indicators.md

```diff
@@ -0,0 +1,60 @@
+---
+published: false # change to true once ready to post
+
+title: "Climate Change API Area Indicators" # project title inside quotes
+excerpt: "A high performance Scala API for calculating climate indicators over geospatial areas" # shows on project list page
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Scala programming experience"
+  - "Python programming experience not required but preferred"
+  - "NetCDF experience not required but preferred"
+tags: # one tag per line – spaces are allowed in tags – follow below format
+  - "scala"
+  - "climate change"
+difficulty: "Hard" # easy, medium, hard – pick one
```

Comment:
Do we have any information about people's interest in easy / medium / hard projects? My guess is that "medium" would be by magic more attractive, and I wouldn't want to scare someone away from a project where we have a lot of recent experience with learning / teaching, specifically:

- DRF -> http4s Andrew just did/is doing, and Chris, Nathan, and I helped with questions he had
- akka-http -> http4s Nathan and I just did, bouncing our inexperience with http4s off each other in the process

So I think, whatever the underlying level of difficulty is, we'll have an unusual amount of empathy for someone new to http4s and for someone trying to map DRF/http4s and akka-http/http4s concepts back and forth, which might bring it down some.

Also I'm suspicious of the "scala is hard" story that we tell ourselves sometimes -- I think it's a bit self-serving by making us seem cooler/more expert-y because we do a hard thing all the time, but it's still just code that either does what we want it to or doesn't, and with mentorship and someone to learn with I think the learning curve can be smoothed considerably.

I won't die on the "it's medium!" hill, but I think we have the recent experience that we can be good mentors around the parts of this project that would make it hard, and that that might make it medium.

Reply:
I think some of the difficulty in this project comes from the size and scope of the expected outcomes. If we rescope some of the project deliverables as optional / stretch goals I think we could scale the difficulty down to medium (https://github.com/azavea/fellowship-program-website/pull/66#discussion_r258495222).

Reply:
I don't have any research concerning how nominal difficulty interacts with self-selection, but it seems like there are plausible arguments in both directions.  If this really is a point of interest, we might consider actually trying to find some research concerning this.

Reply:
I rescoped some of the project goals for phase 2 in https://github.com/azavea/fellowship-program-website/pull/66/commits/1b363182d76f40fca02e2be54ce9f9d2079dc3f6 to cut down on the amount of work and rephrased the remainder as stretch goals, which made me feel comfortable dropping this down to a medium.